programs/cleos/main.cpp

```diff
@@ -2299,9 +2299,10 @@ void get_account( const string& accountName, const string& coresym, bool json_fo
          return ss.str();
       };
 
-      const std::string net_current_usage_str = "    ( " + (res.net_limit.current_used.valid() ? to_pretty_net(*res.net_limit.current_used) : "out of date") + " )";
+      const std::string out_of_date_str = "    ( out of date )";
+      const std::string net_current_usage_str = (res.net_limit.current_used.valid() ? to_pretty_net(*res.net_limit.current_used) : (to_pretty_net( res.net_limit.used ) + out_of_date_str) );
       std::cout << std::fixed << setprecision(3);
-      std::cout << indent << std::left << std::setw(11) << "used:"      << std::right << std::setw(18) << to_pretty_net( res.net_limit.used ) << net_current_usage_str << "\n";
+      std::cout << indent << std::left << std::setw(11) << "used:"      << std::right << std::setw(18) << net_current_usage_str << "\n";
```

Comment:
I don't think this will preserve the alignment expectations that existed before. Can you show example output of `cleos get account` both for the `(out of date)` case and the normal case?

I suggest doing something like the following instead:
```cpp
std::cout << std::fixed << std::setprecision(3);
std::cout << indent << std::left << std::setw(11) << "used:"      << std::right << std::setw(18);
if( res.net_limit.current_used ) {
   std::cout << to_pretty_net(*res.net_limit.current_used) << "\n";
} else {
   std::cout << to_pretty_net(res.net_limit.used) << "    ( out of date )\n";
}
std::cout << indent << std::left << std::setw(11) << "available:" << std::right << std::setw(18) << to_pretty_net( res.net_limit.available ) << "\n";
std::cout << indent << std::left << std::setw(11) << "limit:"     << std::right << std::setw(18) << to_pretty_net( res.net_limit.max ) << "\n";
std::cout << std::endl;
```